src/ui/style.css

```diff
@@ -0,0 +1,225 @@
+@import url('https://fonts.googleapis.com/css2?family=Sora:wght@100;400;600;700&display=block');
+
+.xar-light-mode {
+  --fg: #101010;
+  --bg: #f7f7f7;
+  --background: #ffffff;
+  --inputShadow: inset -1px -7px 7px rgba(255, 255, 255, 0.7),
+    inset 3px 1px 6px rgba(174, 174, 192, 0.2);
+}
+
+.xar-dark-mode {
+  --fg: #f7f7f7;
+  --bg: #101010;
+  --background: #262626;
+  --inputShadow: inset -2px -2px 4px rgb(57 57 57 / 44%),
+    inset 5px 5px 10px rgb(11 11 11 / 50%);
+}
+
+#xar-modal {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: fixed;
+  z-index: 1;
```

Comment:
Should we add a bigger z-index?

Reply:
@shrinathprabhu Suggestion?

Reply:
Something big like 99999, so parent app's elements don't float on top of the modal

Reply:
Updated.